src/pages/home/report/ReportActionsView.js

```diff
@@ -252,7 +252,18 @@ class ReportActionsView extends React.Component {
     updateSortedReportActions(reportActions) {
         this.sortedReportActions = _.chain(reportActions)
             .sortBy('sequenceNumber')
-            .filter(action => action.actionName === 'ADDCOMMENT' || action.actionName === 'IOU')
+            .filter((action) => {
+                // To filter deleted comments out we need check if the action is of type ADDCOMMENT then the message
```

Comment:
NAB but I found this comment to be a bit overly-complex. Maybe something simpler like this would do the trick:

```suggestion
                // Only show non-empty ADDCOMMENT actions or IOU actions
```

Reply:
That is a great resume of what this is doing